extensions/shared/components/audio-player/index.jsx

```diff
@@ -105,6 +114,7 @@ function AudioPlayer( {
 			onPlay && audio.removeEventListener( 'play', onPlay );
 			onPause && audio.removeEventListener( 'pause', onPause );
 			onError && audio.removeEventListener( 'error', onError );
```

Comment:
Is the next step to add store actions for playing and pausing?

Reply:
yes, everything should be controlled by the state tree. Also setting the current time position, among other ones.